package.json

```
@@ -75,7 +75,9 @@
     "postghpages": "rimraf ./dist/ghpages",
     "preghpages.win": "gulp core:bundleLocale && npm run build.ghpages && echo include: ['_default.json', '_contexts.json', '_base.json'] > dist/ghpages/_config.yml",
     "ghpages.win": "npx ngh --dir=dist/ghpages --no-silent=false --message=%npm_package_version%",
-    "postghpages.win": "rimraf ./dist/ghpages"
+    "postghpages.win": "rimraf ./dist/ghpages",
+    "cypress:open": "cypress open",
+    "cypress:run": "cypress run"
```

Est-ce qu'on pourrait utiliser les commandes de notre Angular.json? L'exemple ci-dessous, prend en compte que nous avons changé les noms de commande dans le Angular.json pour `e2e-open` et `e2e-run`.
```suggestion
    "e2e.open": "ng run demo:e2e-open",
    "e2e.run": "ng run demo:e2e-run",
```

---

Good!